WooCommerce/WooCommerceTests/POS/ViewModels/TotalsViewModelTests.swift

```diff
@@ -22,8 +24,37 @@ final class TotalsViewModelTests: XCTestCase {
                               currencyFormatter: .init(currencySettings: .init()),
                               paymentState: .acceptingCard,
                               isSyncingOrder: false)
+        cancellables = []
+    }
+    override func tearDown() {
+        cancellables.forEach {
+            $0.cancel()
+        }
+        cancellables.removeAll()
```

Comment:
TIL: That's an interesting one, while removeAll should be enough it seems safer to also cancel any ongoing subscriptions to avoid any potential timing issue when deallocating 💯